src/containers/DeepLink/LoginRequestInfo/LoginRequestInfo.js

```diff
@@ -97,6 +102,74 @@ const LoginRequestInfo = props => {
     setReq(new primitives.LoginConsentRequest(deeplinkData))
   }, [deeplinkData]);
 
+  const buildAlert = (request) => {
+    return createAlert(
+      request.title,
+      request.data,
+      [
+        {
+          text: 'DECLINE',
+          onPress: () => resolveAlert(false),
+          style: 'cancel',
+        },
+        {
+          text: 'ACCEPT', onPress: () => {
+
+            var _permissions = [];
+            for (let i = 0; i < permissions.length; i++) {
+              _permissions.push(permissions[i]);
+              if (_permissions[i].vdxfkey == request.vdxfkey) {
+                _permissions[i].agreed = true;
+              }
+            }
+            setExtraPermissions(_permissions);
+
+            resolveAlert(true)
+          }
+        },
+      ],
+      {
+        cancelable: true,
+      },
+    )
+  }
+
+
+  useEffect(() => {
+
+    if (req && req.challenge && req.challenge.requested_access) {
+      if (req.challenge.requested_access.length === 1) {
```

Comment:
If a login consent request only contained the primitives.IDENTITY_AGREEMENT requested_access type, it seems this would allow the user to continue through the request without being informed of what they need to agree to. As the identity agreement flow isn't part of the VerusID provisioning flow (as far as I know?), I would recommend removing ID agreement code from the PR for now and reintroducing it in a future PR.